lang/ar.js

```diff
@@ -18,10 +18,10 @@ export default {
 	"components.filter.clearDescription": "مسح عوامل التصفية لـ: {filterName}",
 	"components.filter.clearDescriptionSingle": "مسح عوامل التصفية",
 	"components.filter.loading": "يتم تحميل عوامل التصفية",
-	"components.filter.filterCountDescription": "{number, plural, zero {لم يتم تطبيق عوامل تصفية.} one {تم تطبيق عامل تصفية واحد.} other {{number} من عوامل التصفية التي تم تطبيقها.}}",
+	"components.filter.filterCountDescription": "{number, plural, =0 {لم يتم تطبيق عوامل تصفية.} one {تم تطبيق عامل تصفية واحد.} other {{number} من عوامل التصفية التي تم تطبيقها.}}",
 	"components.filter.filters": "عوامل التصفية",
 	"components.filter.noFilters": "ما من عوامل تصفية متوفرة",
-	"components.filter.searchResults": "{number, plural, zero {ما من نتائج بحث} one {نتيجة بحث واحدة} other {{number} من نتائج البحث}}",
+	"components.filter.searchResults": "{number, plural, =0 {ما من نتائج بحث} one {نتيجة بحث واحدة} other {{number} من نتائج البحث}}",
```

Comment:
TODO: these should be `zero` not `=0`. I've asked in the localization channel how to get this resolved at the source.

Reply:
I get the same result for `zero` and `=0` for `ar` language.  Is one more correct than the other?  I see [ICU Documentation](https://unicode-org.github.io/icu/userguide/format_parse/messages/) referencing `=0`, but then I see [CLDR data](http://unicode.org/Public/cldr/40/) listing `zero`.  Is there another source that is better to reference?

Reply:
It's not going to make a difference in this case. `zero` is really for translators to add themselves in the case where we don't already have `=0`.

Consider the following example in English:
```
one {1 big red dog}
other {{number} big red dogs}
```
This would produce both "0 big red dogs" and "10 big red dogs" from the `other` case. In Arabic, this may not be enough, and they would need to add a `zero` (and possibly `two`, `few` and `many`) case.

```
// Unkown translation quality from Google using Latin characters
// for decipherability but it gets the point across

zero {0 kabirat min alkilab alhamra'}
one {1 kalb 'ahmar kabir}
two {2 kabirat min alkilab alhamra'}
other {{number} kilab hamra' kabira}
```

It's kind of semantic as they could easily add `=0` and get the same result, but my understanding is exact number cases are generally meant to span languages.

```
=2 {You win a special prize!}
one {1 big red dog}
other {{number} big red dogs}
```
That's a more stark example, but the way we generally use it (e.g. "No search results") is basically the same thing.

[This](https://github.com/eemeli/make-plural/blob/master/packages/plurals/plurals.js#L40) is what I usually use to understand the plural/ordinal rules for each language.